### PR TITLE
padraic-oneill-job-tracker-assessment

### PR DESCRIPTION
**Pull request by podoglyph**

### CODE REVIEW

File: spec/features/users/user_can_see_name_of_each_tag_for_that_job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    tag3 = Tag.create(title: "Dog Sitting")
+
+    visit company_job_path(company, job)
+
+    expect(page).to have_content("ESPN")
+    expect(page).to have_content("Developer")
+    expect(page).to have_content("Aerospace")
+    expect(page).to have_content("Mechanical Engineering")
+    expect(page).to_not have_content("Dog Sitting")
+  end
```
Comment (laurenfazah) [inline spec/features/users/user_can_see_name_of_each_tag_for_that_job_spec.rb]: good edge case testing here

File: app/models/job.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ class Job < ActiveRecord::Base
   has_many :tags, through: :job_tags
 
   def count_tags
-
+    self.job_tags.count(:job_id)
   end
```
Comment (laurenfazah) [inline app/models/job.rb]: I like the creativity here for this helper method

**Comment by laurenfazah** [comment]
Overall, great work. By following your commits, it looks like you did a great job of working methodically through the user stories. 

Your feature tests are really well thought out with edge cases and all. Introducing Factory Girl could make for strong refactors there.

Unfortunately, I'm not seeing any model tests. Something like the `count_tags` method on the `Job` model ideally would have been test driven from this angle. Testing the associations between `jobs` and `tags` would have also been nice here.
